src/components/materialization/GenerateButton.tsx

```diff
@@ -124,7 +163,7 @@ function MaterializeGenerateButton({
             const newDraftId = draftsResponse.data[0].id;
             const draftSpec = generateDraftSpec(
                 encryptedEndpointConfig.data,
-                imageTag.imagePath,
+                imageTag ? imageTag.iconPath : '',
```

Comment:
`imageTag` is either a `string` or `undefined`. Since `generateDraftSpec` expects a string as input, an empty string is passed in the event `imageTag` is `undefined`.